src/variorum/Intel/Broadwell_4F.c

```diff
@@ -439,23 +462,26 @@ int fm_06_4f_get_node_power_json(json_t *get_power_obj)
 
 int fm_06_4f_get_node_power_domain_info_json(json_t *get_domain_obj)
 {
-#ifdef VARIORUM_LOG
-    printf("Running %s\n", __FUNCTION__);
-#endif
 
-    json_get_power_domain_info(get_domain_obj, msrs.msr_pkg_power_info,
-                               msrs.msr_dram_power_info,
-                               msrs.msr_rapl_power_unit, msrs.msr_pkg_power_limit);
+    char *val = getenv("VARIORUM_LOG");
+    if (val != NULL && strlen(val) > 0)
+    {
+        printf("Running %s\n", __FUNCTION__);
+    }
+    msrs.msr_dram_power_info,
+    msrs.msr_rapl_power_unit, msrs.msr_pkg_power_limit;
```

Comment:
Incorrectly deleted the original call to ` json_get_power_domain_info`, stray arguments left behind.